src/components/exchange/ExchangeSearch.js

```diff
@@ -68,10 +104,66 @@ const ExchangeSearch = (
     onChangeText?.('');
   }, [ref, onChangeText]);
 
+  const isFetching = false;
+
+  const spinnerRotation = useSharedValue(0, 'spinnerRotation');
```

Comment:
heh, it's some hack I made for the previous version of reanimated. It's no longer needed!